go.mod

```diff
@@ -12,6 +12,7 @@ require (
 	github.com/stretchr/testify v1.5.1
 	golang.org/x/net v0.0.0-20220909164309-bea034e7d591 // indirect
 	golang.org/x/sys v0.0.0-20220915200043-7b5979e65e41 // indirect
+	golang.org/x/text v0.4.0 // indirect
```

Comment:
This was automatically updated from `make generate` regardless of my change

Reply:
This is okay